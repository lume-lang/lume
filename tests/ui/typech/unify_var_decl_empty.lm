fn foo() {
    let arr = std::Array::new();
}
