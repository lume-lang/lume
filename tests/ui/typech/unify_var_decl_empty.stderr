× error[LM4139]: could not infer type argument
   ╭─[unify_var_decl_empty.lm:2:27]
 1 │ fn foo() {
 2 │     let arr = std::Array::new();
   ∶                           ^^^ could not infer type argument T
 3 │ }
   ╰──
   help: try specifying the type arguments explicitly